mesheryctl/internal/cli/root/design/design_test.go

```diff
@@ -37,97 +37,106 @@ func TestPatterncmd(t *testing.T) {
 					ResponseCode: 200,
 				},
 				{
 					Method:       "POST",
 					URL:          testContext.BaseURL + "/api/pattern/deploy",
-					Response:     "pattern.apply.deploy.golden",
+					Response:     "design.apply.deploy.golden",
 					ResponseCode: 200,
 				},
 			},
 			Token:       filepath.Join(fixturesDir, "token.golden"),
 			ExpectError: false,
 		},
 		{
-			Name:             "pattern delete",
-			Args:             []string{"delete", "-f", filepath.Join(fixturesDir, "pattern.golden")},
-			ExpectedResponse: "pattern.delete.output.golden",
+			Name:             "design delete",
+			Args:             []string{"delete", "-f", filepath.Join(fixturesDir, "design.golden")},
+			ExpectedResponse: "design.delete.output.golden",
 			URLs: []utils.MockURL{
 				{
 					Method:       "DELETE",
 					URL:          testContext.BaseURL + "/api/pattern/deploy",
-					Response:     "pattern.delete.golden",
+					Response:     "design.delete.golden",
 					ResponseCode: 200,
 				},
 			},
 			Token:       filepath.Join(fixturesDir, "token.golden"),
 			ExpectError: false,
 		},
-		//commented to pass in workflow run
 		{
-			Name:             "pattern list",
-			Args:             []string{"list"},
-			ExpectedResponse: "pattern.list.output.golden",
+			Name:             "design view",
+			Args:             []string{"view", "kumatest"},
+			ExpectedResponse: "design.view.kuma.output.golden",
 			URLs: []utils.MockURL{
 				{
 					Method:       "GET",
-					URL:          testContext.BaseURL + "/api/pattern",
-					Response:     "pattern.list.golden",
+					URL:          testContext.BaseURL + "/api/pattern?search=kumatest",
+					Response:     "design.view.kuma.golden",
 					ResponseCode: 200,
 				},
-			},
-			Token:       filepath.Join(fixturesDir, "token.golden"),
-			ExpectError: false,
-		},
-		{
-			Name:             "pattern view",
-			Args:             []string{"view", "kumatest"},
-			ExpectedResponse: "pattern.view.kuma.output.golden",
-			URLs: []utils.MockURL{
 				{
 					Method:       "GET",
-					URL:          testContext.BaseURL + "/api/pattern?search=",
-					Response:     "pattern.view.kuma.golden",
+					URL:          testContext.BaseURL + "/api/pattern?page_size=10000",
+					Response:     "view.design.api.response.golden",
 					ResponseCode: 200,
 				},
 			},
 			Token:       filepath.Join(fixturesDir, "token.golden"),
 			ExpectError: false,
 		},
 		{
-			Name:             "pattern view with ID",
+			Name:             "design view with ID",
 			Args:             []string{"view", "4o7fbc9b-708d-4396-84b8-e2ba37c1adcc"},
-			ExpectedResponse: "pattern.id.view.output.golden",
+			ExpectedResponse: "design.id.view.output.golden",
 			URLs: []utils.MockURL{
 				{
 					Method:       "GET",
-					URL:          testContext.BaseURL + "api/pattern/",
-					Response:     "pattern.id.view.golden",
+					URL:          testContext.BaseURL + "/api/pattern/4o7fbc9b-708d-4396-84b8-e2ba37c1adcc",
```

Comment:
if we do a delete on that id the test will fail so not sure it is really relevant